build.sbt

```diff
@@ -162,7 +162,8 @@ lazy val commonSettings =
       //here we add dependencies that we want to have fixed across all modules
       dependencyOverrides ++= Seq(
         //currently Flink (1.9) uses 1.8.2 Avro version
```

Comment:
This comment is now depricated?

Reply:
1.9 -> deprecated, but even now (Flink 1.11) 1.8.2 is used. It will remain so for a LONG time I guess (hive problems etc...)